### PR TITLE
Add actuator docs to github.

### PR DESCRIPTION
**Pull request by surfnerd**
### Proposed change(s)

Add a small section on Actuators to github.

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [x] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: docs/Learning-Environment-Design-Agents.md [hunk 1]
```diff
@@ -647,6 +649,29 @@ Notes:
...
+
+Like the `ISensor` interface, the `IActuator` interface is intended for advanced users.
+
+The `ActuatorComponent` abstract class is used to create the actual `IActuator` at
+runtime. It must be attached to the same `GameObject` as the `Agent`, or to a
+child `GameObject`.  Actuators and all of their data structures are initialized
+during `Agent.Initialize`.  This was done to prevent an unexpected allocations at runtime.
+
+You can find an example of an `IActuator` implementation in the `Basic` example scene.
+**NOTE**: you do not need to adjust the Space Size in the Agent's
```
Comment (chriselion) [inline docs/Learning-Environment-Design-Agents.md:668]:
```suggestion
**NOTE**: you do not need to adjust the Actions in the Agent's
```
This is just `Actions` (with `Continuous Actions` and `Discrete Branches`) in the inspector now.

File: docs/Learning-Environment-Design-Agents.md [hunk 2]
```diff
@@ -507,12 +508,13 @@ setting the State Size.
 - Use as few rays and tags as necessary to solve the problem in order to improve
   learning stability and agent performance.
 
-## Actions
+## Actions and Actuators
```
Comment (chriselion) [inline docs/Learning-Environment-Design-Agents.md:511]: I think this breaks the existing table of contents (`- [Actions](#actions)`). Maybe keep it as it is; I'll try to propose an edit to the ToC.

Comment (surfnerd) [reply docs/Learning-Environment-Design-Agents.md:511]: Oh, I see.  I messed up that merge.

Comment (surfnerd) [reply docs/Learning-Environment-Design-Agents.md:511]: i can fix it real quick

File: docs/Learning-Environment-Design-Agents.md [hunk 3]
```diff
@@ -18,6 +18,7 @@
   - [Raycast Observations](#raycast-observations)
     - [RayCast Observation Summary & Best Practices](#raycast-observation-summary--best-practices)
 - [Actions](#actions)
+  - [Actions and Actuators](#actions-and-actuators)
```
Comment (chriselion) [inline docs/Learning-Environment-Design-Agents.md:21]:
```suggestion
```

File: docs/Learning-Environment-Design-Agents.md [hunk 4]
```diff
@@ -647,6 +649,29 @@ Notes:
 - You cannot mask all the actions of a branch.
 - You cannot mask actions in continuous control.
 
+
+#### IActuator interface and ActuatorComponents
```
Comment (chriselion) [inline docs/Learning-Environment-Design-Agents.md:653]:
```suggestion
### IActuator interface and ActuatorComponents
```
(same level as `Discrete Actions`)

Comment (chriselion) [reply docs/Learning-Environment-Design-Agents.md]: Arg, couldn't get the multi-line suggestion working, but add a line for this under Discrete Actions in the ToC

**Review by chriselion — APPROVED** [review]
Looks good once the Table of Contents is fixed up.

**Pull request merged**